src/views/Questionnaire/MyQuestionnaireAppraisee.vue

```diff
@@ -1,35 +1,56 @@
+          <p class="my-questionnaire-appraisees__content__questionnaire-description">
             {{ currentQuestionnaire.description }}
           </p>
-          <div class="questionnaire-date">
-            <div class="placeholder-start-date">
-              Start Date : {{ computedDate(currentQuestionnaire.startDate) }}
+          <div class="my-questionnaire-appraisees__content__questionnaire-date">
+            <div class="my-questionnaire-appraisees__content__questionnaire-date__placeholder-start-date">
+              Start Date :
+              <span class="green">
+                <br>
+                {{ computedDate(currentQuestionnaire.startDate) }}
+              </span>
             </div>
-            <div class="placeholder-due-date">
-              Due Date : {{ computedDate(currentQuestionnaire.dueDate) }}
+            <div class="my-questionnaire-appraisees__content__questionnaire-date__placeholder-due-date">
+              Due Date :
+              <span class="red">
+                <br>
+                {{ computedDate(currentQuestionnaire.dueDate) }}
+              </span>
             </div>
           </div>
         </div>
-        <div v-if="!this.$route.params.appraiseeId" class="my-appraisees-list-containner">
-          <div class="title-appraisee">
+        <div v-if="!this.$route.params.appraiseeId" class="my-questionnaire-appraisees__content__appraisees-list">
+          <div class="my-questionnaire-appraisees__content__appraisees-list__title-appraisee">
             <p><strong>Appraisee</strong></p>
           </div>
-          <QuestionnaireParticipantCard v-for="appraisee in myListAppraisees"
-                                        :key="appraisee.id"
-                                        :name="appraisee.name"
-                                        :avatar="appraisee.avatar"
-                                        :role="appraisee.role"
-                                        :university="appraisee.university"
-                                        :batch="appraisee.batch.name"
-                                        v-on:click="goToInputQuestionnaireAnswer(appraisee)"
-          ></QuestionnaireParticipantCard>
+          <div class="my-questionnaire-appraisees__content__appraisees-list__appraisee-card-container">
+            <QuestionnaireParticipantCard v-for="appraisee in myListAppraisees"
+                                          :key="appraisee.id"
+                                          :name="appraisee.name"
+                                          :avatar="appraisee.avatar"
+                                          :role="appraisee.role"
+                                          :university="appraisee.university"
+                                          :batch="appraisee.batch.name"
+                                          v-on:click="goToInputQuestionnaireAnswer(appraisee)"
```

Comment:
Can be changed `@click` instead of `v-on`

Reply:
Resolved, please check again